src/Javinizer/Public/Get-JVData.ps1

```diff
@@ -226,10 +236,10 @@ function Get-JVData {
                     Start-ThreadJob -Name "jvdata-Dmm" -ThrottleLimit $throttleLimit -ScriptBlock {
                         Import-Module $using:jvModulePath

```

Comment:
Can you change the threadjob name to `jvdata-DmmJa`? Just realized this is inconsistent with the others.